.gitignore

```diff
@@ -0,0 +1,2 @@
+# This file is laid down by the ansible script that builds the virtual machine.  Make it ignored so we don't run into issues about not having a clean repository (which makes ansible unhappy)
+.env.isamples_inabox
```

Comment:
This file is laid down in place by the ansible script, so ignore it if on the fs.